### PR TITLE
adds some description for adding dist extensions

### PR DESCRIPTION
**Pull request by mikebrow**
from discussion on the OCI call... [see extension proposal](https://hackmd.io/El8Dd2xrTlCaCG59ns5cwg?edit)

here's a description in the readme, acknowledging that we are moving into a stage for prototyping extensions to the distribution spec/api. Details to follow as we move along with the first extension, the extension API itself.

@sajayantony @stevvooe @vbatts 

related:
https://github.com/opencontainers/distribution-spec/pull/111
https://hackmd.io/8vXj0k1HSIGF2kauCgnroA

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,6 +55,23 @@ For general questions about OCI, please see the [FAQ on the OCI site](https://ww
...
+
+The Distribution Specification Project is including, herein, a process and API for prototyping
+and testing extensions to the Distribution API.
+
+We invite contributions, comments, and reviews to these extensions.
+These extensions will only move forward with significant support from registies, registry clients, and users.
+
+Disclaimer: Unless stated otherwise, extension prototypes are subject to change. Extensions
+should NOT be considered as final or approved official extensions to the OCI Distribution API until they
+have been included in an OCI Distribution Specification.
```
Comment (jdolitsky) [inline README.md:68]: It seems like this is the case now. Should we close this PR @mikebrow ?

**Comment by jdolitsky** [comment]
We probably actually want to point to this doc within `spec.md`. WDYT?

File: README.md (hunk 1 above)
Comment (mikebrow) [reply README.md:68]: I need to update this PR to point to https://github.com/opencontainers/distribution-spec/blob/main/extensions/README.md

Comment (jdolitsky) [inline README.md:68]:
```suggestion
Please see [here](https://github.com/opencontainers/distribution-spec/blob/main/extensions/README.md) for more details.
```

File: README.md [hunk 2]
```diff
@@ -55,6 +55,23 @@ For general questions about OCI, please see the [FAQ on the OCI site](https://ww
...
+
+## Extensions API - (status: WIP)
+
+The Distribution maintainers recognize that an API may be necessary, or at least potentially advantageous, for specifying
+extensions to the Distribution API. Thus, we propose to add an [Extensions API for Distribution](EXTENSIONS.md).
+
```
Comment (jdolitsky) [inline README.md:74]:
```suggestion
```

**Review by jdolitsky — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -55,6 +55,16 @@ For general questions about OCI, please see the [FAQ on the OCI site](https://ww
 
 The [GitHub milestones](https://github.com/opencontainers/distribution-spec/milestones) lay out the path to the future improvements.
 
+# Extensions
+
+The Distribution Specification Project is including, herein, a process and API for prototyping
+and testing extensions to the Distribution API.
+
+We invite contributions, comments, and reviews to these extensions.
+These extensions will only move forward with significant support from registies, registry clients, and users.
```
Comment (jdolitsky) [inline README.md:64]:
```suggestion
These extensions will only move forward with significant support from registries, registry clients, and users.
```

File: README.md [hunk 4]
```diff
@@ -55,6 +55,16 @@ For general questions about OCI, please see the [FAQ on the OCI site](https://ww
 
 The [GitHub milestones](https://github.com/opencontainers/distribution-spec/milestones) lay out the path to the future improvements.
 
+# Extensions
+
+The Distribution Specification Project is including, herein, a process and API for prototyping
```
Comment (jdolitsky) [inline README.md:60]:
```suggestion
The Distribution Specification Project is includes a process and API for prototyping
```

**Review by jdolitsky — DISMISSED** [review]

File: README.md [hunk 5]
```diff
@@ -55,6 +55,16 @@ For general questions about OCI, please see the [FAQ on the OCI site](https://ww
 
 The [GitHub milestones](https://github.com/opencontainers/distribution-spec/milestones) lay out the path to the future improvements.
 
+# Extensions
+
+The Distribution Specification Project is includes a process and API for prototyping
```
Comment (jdolitsky) [inline README.md:60]:
```suggestion
The Distribution Specification Project includes a process and API for prototyping
```

**Review by jdolitsky — APPROVED** [review]

**Review by jonjohnsonjr — APPROVED** [review]

**Pull request merged**